vigetlabs/wordpress-site-starter
# [#67] Header navigation

# Summary

This styles the header navigation. Removed the WP default a11y as it was not fully 100% a11y and built my own with AlpineJS. Because we can't access some of the navigation HTML I had to inject the Alpine with JS onto the DOM. 

The dark version will just change if we add `dark` to the parent div in the parts. 

The styles are pretty basic allowing us to customize per client. 

## Issues

* #67 

## Testing Instructions

1. Add navigation items in the admin. 
2. Test out that navigation on desktop and mobile. 
3. Test with keyboard and screen reader to make sure everything is a11y. 

## Screenshots
| | |
|--------|--------|
| <img width="1080" alt="Screenshot 2024-05-21 at 8 13 18 AM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/91974372/202dbb1b-ee08-40a1-9a5d-258cbfdfe355"> | <img width="1081" alt="Screenshot 2024-05-21 at 8 13 04 AM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/91974372/333abf38-ed0d-4c8a-b65d-8581120aca85"> |
| <img width="511" alt="Screenshot 2024-05-21 at 8 14 34 AM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/91974372/6cebcfa3-0b4b-4b90-96f6-9cecaac35e18"> | <img width="510" alt="Screenshot 2024-05-21 at 8 15 02 AM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/91974372/bf741257-a1bb-4fe2-86ff-27eb242db87b"> |



https://github.com/vigetlabs/wordpress-site-starter/assets/91974372/bd44aba4-419a-4f7c-94b5-f510e879f563
